packages/airnode-protocol/contracts/whitelist/WhitelistRoles.sol

```diff
@@ -16,6 +16,10 @@ contract WhitelistRoles is RoleDeriver, AccessControlClient, IWhitelistRoles {
     //     └── (4) Indefinite whitelister
     // Their IDs are derived from the descriptions below. Refer to
     // AccessControlRegistry for more information.
+    // To clarify, the root role of the manager is the admin of (1), while (1)
+    // is the admin of (2), (3) and (4). So (1) is more of a "contract admin",
+    // while the `adminRole` used in AccessControl and AccessControlRegistry
+    // refers to a more general adminship relationship between roles.
```

Comment:
This is to address API3-08-2. I tried to come up with a different than "admin" here, but I think it's actually apt and some additional documentation is probably enough to clear away potential confusion.